src/migrations/20230915094540-dependent-flags.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+exports.up = function (db, callback) {
+    db.runSql(
+        `
+          ALTER TABLE feature_strategies ADD COLUMN IF NOT EXISTS dependencies JSONB DEFAULT '[]'::jsonb NOT NULL;
```

Comment:
I am not sure if we want the dependencies on the strategies themselves. Is that what you plan to do here?

Reply:
this is still a draft pr (updated the status to avoid confusion). I'm experimenting with different models and this one can be either on the strategy or feature_environment or feature. We can have a chat on Monday.

Reply:
BTW I have full working e2e spike on my machine I can demo to you next week. With dependencies on strategies you can express: if parent flag resolved to variantA resolve child flag to variantB, if parent flag resolved to variant C resolve child flag to variant D.